php/deepgram-transcribe-video/index.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+function deepgramTranscribeVideo($fileUrl) {
+
+    $apiKey = '<API-KEY>';
+    $apiSecret = '<API-SECRET>';
+    $requestUrl = 'https://api.deepgram.com/v2/speech';
+
+    $requestData = array(
+        'url' => $fileUrl,
+    );
+
+    $requestOptions = array(
+        'http' => array(
+            'method' => 'POST',
+            'header' => "Content-Type: application/json\r\n" .
+                        "Authorization: Basic " . base64_encode("$apiKey:$apiSecret") . "\r\n",
+            'content' => json_encode($requestData),
+        ),
+    );
+
+    $context = stream_context_create($requestOptions);
+    $response = file_get_contents($requestUrl, false, $context);
+
+    return json_decode($response, true);
+}
+
+?>
```

Comment:
This function does not conform to the interface for an open runtimes function. For example, refer to this: https://github.com/open-runtimes/examples/blob/ac633af06673b1de963b19d16e9a041f941984a0/php/send_message/index.php. 

Please make sure to add error handling so to that the response is `{"success": false, "message": "<descriptive error message>"}`